packages/styles/scss/fonts/README.md

```diff
@@ -0,0 +1,134 @@
+# `scss/fonts`
+
+The `scss/fonts` folder provides a way to incorporate IBM Plex into your
+project. Below is a table of what fonts are currently supported from IBM Plex in
+Carbon:
+
+| Font                      | Available | Entrypoint                    |
+| ------------------------- | --------- | ----------------------------- |
+| IBM Plex Mono             | ✅        | `scss/fonts/mono`             |
+| IBM Plex Sans Arabic      | ✅        | `scss/fonts/sans-arabic`      |
+| IBM Plex Sans Devanagari  | ✅        | `scss/fonts/sans-devanagari`  |
+| IBM Plex Sans Hebrew      | ✅        | `scss/fonts/sans-hebrew`      |
+| IBM Plex Sans KR          |           |                               |
+| IBM Plex Sans JP          |           |                               |
```

Comment:
@tay1orjones we ran into two problems when implementing:

- JP was not included in the current Plex beta
- These fonts support hinted and unhinted variants which complicated an already pretty complicated resolver logic lol, felt better to add later instead of all in one
- Like you said, the font files and resulting package can be HUGE